ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -161,9 +161,36 @@ public void onAdStart(String placementId) {
         }
 
         @Override
+        @Deprecated
         public void onAdEnd(String placementId, boolean completed, boolean isCTAClicked) {
+
+        }
+
+        @Override
+        public void onAdEnd(String placementId) {
+            if (mPendingRequestBanner && mVungleListener != null) {
```

Comment:
we should not check for mPendingRequestBanner here, it can happen on destroy call

Reply:
Updated here: https://github.com/Vungle/googleads-mobile-android-mediation/pull/35/files#diff-059c77ef8b7787eb50e9a78d86099aeaR95